enterprise/core-edge/src/test/java/org/neo4j/coreedge/raft/log/ConcurrentStressIT.java

```diff
@@ -141,14 +148,16 @@ private void write( RaftLog raftLog )
         }
     }
 
+    private static final CharSequence CHARS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
```

Comment:
Not anymore after you limit the set of characters used, but we should really fix it, because the default on windows isn't bijective and using CP-1252 I think.